app/domain/audits/find_content.rb

```diff
@@ -0,0 +1,23 @@
+module Audits
+  class FindContent
+    def self.call(filter = Filter.new)
+      scope = Content::Query.new
+                .page(filter.page)
+                .organisations(filter.organisations, filter.primary_org_only)
+                .document_type(filter.document_type)
+                .after(filter.after)
+                .theme(filter.theme_id)
+                .scope
+
+      return scope if filter.audit_status.blank?
+
+      if filter.audit_status.to_sym == :audited
```

Comment:
I have addressed this in 9d245286b7cf774a723f0b13aeb43b326749ff70
I have extract a nil object to deal with the `nil` | `blank` situation. It also make `FindContent` agnostic of the list of values for audit_status